### DISCUSSION TITLE
German Translation Saaatgutbehälter --) Saatgutbehälter

### DISCUSSION THREAD

**Discussion by Deprimiaut** [FakeFishGames/Barotrauma#14724] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

The seed bag is not written Saaatgutbehälter.
There is one A too many. It is spelled Saatgutbehälter.

### Reproduction steps

*Set the translation to German
*Looking at the seed bag

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.5.9.2 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! We've fixed this now, fix coming up in the next update.
